src/reactSwipe.js

```diff
@@ -85,7 +86,9 @@ class ReactSwipe extends Component {
       <div ref="container" id={id} className={`react-swipe-container ${className}`} style={style.container}>
         <div style={style.wrapper}>
           {React.Children.map(children, (child) => {
-            return React.cloneElement(child, {style: style.child});
+            return React.cloneElement(child, {
+              style: child.props.style ? objectAssign(child.props.style, style.child) : style.child
```

Comment:
@shoaibkalsekar there's no need to check, 'cause `style` is in default props - https://github.com/voronianski/react-swipe/blob/gh-pages/src/reactSwipe.js#L28